project_template/go.sh

```diff
@@ -8,22 +8,30 @@ EOF
 
 set -e
 
+# Detect if this script is run within an existing virtualenv and abort
+if [[ $VIRTUAL_ENV ]]; then
+    echo "ERROR: $0 will not work properly if run inside a venv; deactivate '`basename $VIRTUAL_ENV`'"
+    exit 1
+fi
+
 # Get absolute project directory from the location of this script.
 # See: http://stackoverflow.com/a/4774063
 export ICEKIT_PROJECT_DIR=$(cd $(dirname "${BASH_SOURCE[0]}"); pwd -P)
 
-# Create virtualenv.
-if [[ ! -d "$ICEKIT_PROJECT_DIR/var/venv/bin/pip" ]]; then
-    virtualenv "$ICEKIT_PROJECT_DIR/var/venv"
+export ICEKIT_VENV="$ICEKIT_PROJECT_DIR/var/venv-icekit-dev"
+
+# Create local (non-Docker) virtualenv.
+if [[ ! -d "$ICEKIT_VENV/bin/pip" ]]; then
```

Comment:
I think this needs to be `! -f` or `! -x`, or leave it `! -d` and drop `/bin/pip` from the path. Probably the latter, since we're no longer mixing the user scheme and virtualenv directories.